proposals/evaluation/uc-3-inheritance.md

```diff
@@ -176,14 +209,18 @@ Todo: How does a client create a new acl if one does not exist before?
   acl:mode acl:Read, acl:Write .
 ```
 
+As soon this is created, the old ACR will no longer be authoritative. 
+
 ### WAC+ relaxing acl:default
 
+If one were to want the `</weekly-status/2021-04-28/>` ACR to continue to use `</weekly-status/.acl>` then one could make the `acl:default` more flexible as follows.
+
 According to the [ACL ontology definition as of July 2021](https://github.com/solid/authorization-panel/pull/216#discussion_r665338497), the `acl:default` predicate is only effective in statements where the current container is the object; that is, the resource `</weekly-status/.acl>`, which is the direct effective access control list of `</weekly-status/>`, can only target that directory in statements using `acl:default`.
 
 However, if WAC's use of `acl:default` were to be relaxed as described in [issue 191](https://github.com/solid/authorization-panel/issues/191), then one could rely on the effective access control resource discovery mechanism and augment the content of `</weekly-status/.acl>`:
 
 ```Turtle
-# Resource: </weekly-status/.acl>
+# Resource: </weekly-status/2021-04-28/.acl>
```

Comment:
```suggestion
# Resource: </weekly-status/.acl>
```

We're describing the case where `acl:default` is relaxed and can target any container that end up with the Access control resource as effective ACL.